GameDesign/Assets/Scripts/Managers/PointsManager.cs

```diff
@@ -34,17 +41,17 @@ public static PointsManager Instance
 
     private void Awake()
     {
-        if (instance != null && instance != this)
+        if (PointsManager.Instance != null && PointsManager.Instance != this)
         {
             Destroy(this);
         }
         else
         {
             instance = this;
-            DontDestroyOnLoad(gameObject);
+
+            calculateMaxPoints();
```

Comment:
It could happen that at this point the dictionaries haven't been filled out yet by the Level Monobehaviour. We need to find a way to use the SanityCheck (that I know is not on master yet) or a kind of event to failsafe this.